### PR TITLE
Randolph Glacier Inventory Dataset

### PR DESCRIPTION
**Pull request by pradh**
Related to issue https://github.com/datacommonsorg/data/issues/601.

### CODE REVIEW

File: scripts/glims/rgi/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Scripts to import [RGI 6.0](https://www.glims.org/RGI/rgi60_dl.html) attributes
+data into DC.
+
+The downloaded CSVs are available
+[here](https://www.glims.org/RGI/rgi60_files/00_rgi60_attribs.zip).
+
+## Run
+
+1. Download the files to `scratch/`.
```
Comment (beets) [inline scripts/glims/rgi/README.md:11]: maybe add a .gitignore for `scratch/`?

**Review by beets — APPROVED** [review]
nice!

File: scripts/glims/rgi/README.md (hunk 1 above)
Comment (pradh) [reply scripts/glims/rgi/README.md:11]: Done.

Also added unit tests.

**Review by pradh — COMMENTED** [review]
Thanks for the review!

**Pull request merged**